src/lib/utils.ts

```diff
@@ -5,11 +5,17 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import * as Stream from 'stream';
```

Comment:
I think all the changes to this general `utils` module are stream-specific and should go in the new `log-stream-utils` module?